include/s3select_functions.h

```diff
@@ -46,7 +46,8 @@ class s3select_functions : public __clt_allocator {
             {"extract", s3select_func_En_t::EXTRACT},
             {"dateadd", s3select_func_En_t::DATE_ADD},
             {"datediff", s3select_func_En_t::DATE_DIFF},
-            {"utcnow", s3select_func_En_t::UTCNOW}
+            {"utcnow", s3select_func_En_t::UTCNOW},
+            {"charlength", s3select_func_En_t::CHAR_LENGTH}
```

Comment:
https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-string.html#s3-glacier-select-sql-reference-char-length

CHAR_LENGTH and CHARACTER_LENGTH are synonyms.